lib/services/snap.js

```diff
@@ -11,7 +11,7 @@ module.exports = {
       service : 'snap',
       build : env.SNAP_PIPELINE_COUNTER,
       commit : env.SNAP_COMMIT,
-      branch : env.SNAP_BRANCH,
+      branch : env.SNAP_BRANCH || env.SNAP_UPSTREAM_BRANCH,
```

Comment:
on a PR to master, this would resolve to be master, which would result (if I'm not mistaken) in the coverage reports to override what is in master and I don't believe that's what we want. I may be simply misunderstanding how codecov works though...